### DISCUSSION TITLE
multiple ip listening

### DISCUSSION THREAD

**Discussion by LiveG24** [rejetto/hfs#772] [Ideas]
hello
is possible to have hfs listen on localhost for admin panel
but also on one of the lan ip ?

if I choose the lan ip, then I am forced to create admin user and use lan ip to access admin panel

thank you 🤓

**Comment by rejetto** [comment]
nope, you cannot pick 2 ips, at the moment.
the point is to block some ips?

**Comment by LiveG24** [comment]
the point would be to have the hfs to serve only on one of multiple ip's
while mantaining the localhost admin panel password free

**Comment by rejetto** [comment]
then you can use admin-panel / options / blocked ip
to get basically the same

**Comment by LiveG24** [comment]
thank you, will try
